File: Length of Last Word.js
// https://leetcode.com/problems/length-of-last-word/description/
// 58. Length of Last Word

/**
 * @param {string} s
 * @return {number}
 */
// var lengthOfLastWord = function(s) {
//     let text = s;
//     text = text.trim();
//     text = text.split(" ");

//     return text[text.length-1].length

// };
var lengthOfLastWord = function(s) {
    return s.trim().split(" ")[s.trim().split(" ").length-1].length
};

File: Length of Last Word.py
# https://leetcode.com/problems/length-of-last-word/?envType=daily-question&envId=2024-04-01
# Length of Last Word

class Solution:
    def lengthOfLastWord(self, s: str) -> int:
        
        # s = s.strip()

        # count = 0

        # for char in s:
        #     if char == " ":
        #         count = 0
        #     else:
        #         count += 1

        # return count

        return len(s.strip().split()[-1])